CMakeLists.txt

```diff
@@ -333,6 +334,34 @@ endif()
 
 # TODO: Check if ADIOS2 is parallel when openPMD_HAVE_MPI is ON
 
+# External library: libcxxwrap-julia
+if(openPMD_USE_JULIA STREQUAL AUTO)
+    find_package(JlCxx 0.8.3)
+    if(JlCxx_FOUND)
+        set(openPMD_HAVE_JlCxx TRUE)
+    else()
+        set(openPMD_HAVE_JlCxx FALSE)
+    endif()
+    if(JULIA_FOUND)
+        set(openPMD_HAVE_JULIA TRUE)
+    else()
+        set(openPMD_HAVE_JULIA FALSE)
+    endif()
+elseif(openPMD_USE_JULIA)
+    find_package(JlCxx 0.8.3 REQUIRED)
+    set(openPMD_HAVE_JULIA TRUE)
+    set(openPMD_HAVE_JlCxx TRUE)
```

Comment:
We use the `openPMD_USE_...` variables to express requests by the user.
The `openPMD_HAVE_...` variables are to express how we solved that request with the given environment.

Since we can implement Julia bindings only when `JlCxx` was found, we only need a single `openPMD_HAVE_JULIA` variable` for the rest of the control below.

Reply:
Done.